components/FilterTabType/FilterTabType.js

```diff
@@ -30,7 +30,9 @@ export default class FilterTabType extends PureComponent<void, Props, void> {
                 style={styles.accountIssuesType}
                 onPress={onPress}
             >
-                <Text style={styles.accountIssuesTypeText}>{title}</Text>
+                <Text style={styles.accountIssuesTypeText}>
+                    {title.replace(/\s\s/g, '\n')}
```

Comment:
Not very good, but if you write this: "Review\nrequests" and pass to `title` - does not work, no line break.
Can there be another solution?